bearer/bearer.go

```diff
@@ -405,6 +415,8 @@ func (b Token) Issuer() user.ID {
 // Returns zero [user.ID] if Token is unsigned or key has incorrect format.
 //
 // See also [Token.SigningKeyBytes].
+//
+// Deprecated: use [Token.Issuer] instead.
```

Comment:
Belongs to c96853a381df376fb08f624bb9d7e9e8da371417.

Reply:
not really. c96853a381df376fb08f624bb9d7e9e8da371417 just adds methods to set/get new field. This does not affect `ResolveIssuer`. While current commit introduces issuer's signing method recommended in most cases, making issuer resolving redundant

Reply:
one commit after https://github.com/nspcc-dev/neofs-sdk-go/pull/564#discussion_r1513048024